comments/views.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+
+from foods.views import CategoryDetail
+from .models import Comment
+from .serializers import CommentSerializer, CommentDetailSerializer
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class CommentList(ListCreateAPIView):
+    serializer_class = CommentSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def perform_create(self, serializer):
```

Comment:
why have you created a create method in comments lists?

Reply:
Dạ chổ ni ghi dè lạ cái để mình nhận user_id trực tiếp từ token thay vì  mình truyền user_id trong body anh